### ISSUE TITLE
Misleading in the synthesis of electric wire cutters and electric flints

### ISSUE THREAD

**Issue by Mobius0526** [GTNewHorizons/GT-New-Horizons-Modpack#14854]
### Your Pack Version

2.4.0

### Your Server

SP

### Java Version

Java 17

### Type of Server

None

### Your Expectation

。

### The Reality

![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/62875887/ba05e11a-8dee-4581-8bc4-92ab6d42ab22)
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/62875887/3b83edd6-a0fa-46a1-9960-b991fb5d4510)
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/62875887/8190a7ea-be06-403f-9e3f-10ae37e8e11a)
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/62875887/74178448-faee-4655-9a49-ba84a87160db)
For example, a synthetic tungsten steel electric cutting machine requires the use of any wire cutting pliers，
However, based on the formula found, I believe that synthetic tungsten steel electric wire cutters require tungsten steel wire cutters, and tungsten steel wire cutters will be consumed
My description may be confusing because I don't know English. Just open the game and check the formula of the electric wire cutters and other formulas that require wire cutters to understand what the problem is

### Your Proposal

fix it

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by LazyFleshWasTaken** [comment]
Yup, Electric wire cutters, electric butcher knives, and whatever else that wants a tool to be made, I'll continue with the wire cutters, for example, shadow metal electric wire cutters show that they need a shadow metal wire cutter in the recipe, but that cutter can in fact be of any material, and only the plate matters. THe same applies for the electric butcher knife and pyromatic.
